cg/meta/demultiplex/utils.py

```diff
@@ -144,7 +144,7 @@ def get_q30_threshold(sequencer_type: Sequencers) -> int:
     return FLOWCELL_Q30_THRESHOLD[sequencer_type]
 
 
-def get_sample_sheet_path(
+def get_sample_sheet_path_from_flow_cell_dir(
     flow_cell_directory: Path,
     sample_sheet_file_name: str = DemultiplexingDirsAndFiles.SAMPLE_SHEET_FILE_NAME,
 ) -> Path:
```

Comment:
I think we only ever need this in the `FlowCellDirectoryData` class. Our current flow is 
Copy flow cell from NAS -> Create sample sheet -> add sample sheet to houskeeper -> any other flow. 

So if a flow cell does not have a sample sheet in housekeeper it should fail. I think it is safe to centralise the samplesheet to the `FlowCellDirectoryData` and always fetch it in that way.

Reply:
Yes, I don't want to change any existing functionality in this PR though. The intention is to just replace calls to `get_sample_sheets_from_latest_version` with `get_sample_sheet_path`.

Reply:
Sounds good to do this in an upcoming PR 👍

Reply:
👍